.circleci/config.yml

```diff
@@ -477,36 +485,43 @@ workflows:
           k8s_version: "v1.13.0"
           requires:
             - build
+            - helm_check
       - k8s_integration_tests:
           name: "k8s_v1.12.0_integration_tests"
           k8s_version: "v1.12.0"
           requires:
             - build
+            - helm_check
```

Comment:
Why do the k8s integration tests require the helm check?

Reply:
Since the helm test would be included in the k8s test jobs, my thought was that the helm chart should be up-to-date and pass lint.

On a related note, do we want to run `helm_check` for every build, or only if `deployments/k8s` has changes?

Reply:
It seems pretty clear to me that there are no external dependencies for the Helm chart outside of `deployments/k8s` so I think it is safe to skip it.

If making the helm_check a requirement of the k8s test jobs means they take longer to start up, I would rather them not be a requirement, but just run along side the k8s jobs, mainly just to keep the finish time on the k8s jobs as fast as possible.

Reply:
`helm_check` currently only takes ~15 seconds, so it won't add any time since the k8s tests need to wait ~5 minutes for the build job to complete anyways.

I'll update `helm_check` to only run if `deployments/k8s` has changes.

Reply:
ok cool